hyperledger/indy-plenum
# INDY-1304: Added tests that show problems with current view change [test #3]

Tests that show problems with current view change.
#3

Test case:
- disable normal view change to make tests deterministic
- delay commits for all nodes except node X
- send request
- check ordered transaction in node X
- start view_change
- check end of view change for all nodes
- switch off commits' delay
- get reply (means that request was ordered in all nodes)
- repeat

Expected result with correct view change: transactions should be ordered normally
Expected result with current view change: node X can't finish second transaction
